content/post/articles/import_article_to_medium.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Importing OSS Cameroon blog post to Medium."
+date: 2021-07-15T00:15:17+08:00
+images: []
+categories: [tutorial, blogging]
+tags: []
+authors: [EGG]
+---
+
+Welcome to OSS Cameroon! from today on, we won't have any excuse not to gain more visibility by taking advantage of [*Medium*](https://medium.com) blogging platform. 
+Because the process of importing an existing article to [*Medium*](https://medium.com/) is quite straightforward... Let's ✂➡🏃💨.
+
+***Please do not try an import any publication/article/blog/story on which you don't own  publication rights!!***
+
+## Prerequisite
+Obviously at this point, we should know that generally one needs to have an account if they intend to publish on any platform, especially if they expect some rights on it. The steps to import an article on *Medium* are the following:
+1. Create/have a free account on *Medium* site, 
+2. Have the article already posted on *blog.osscameroon.com*
+3. On your desktop browser, log into the *Medium* account, in this case *OSSCameroon's* page.
+4. Click on the profile picture->***Stories***
+<img src="/images/how_to_import_to_medium/profile_picture.jpg" alt="Medium profile pciture"  width="200" height="300"/>
```

Comment:
If you want images to be handled by hugo you might want to use the [figure](https://gohugo.io/content-management/shortcodes/#figure)  shortcode.
You can use what I did on this PR as example https://github.com/osscameroon/osscameroon-blog/pull/24